docs/cli-arguments.md

```diff
@@ -37,13 +37,15 @@ Usage of ./kube-state-metrics:
       --log_backtrace_at traceLocation              when logging hits line file:N, emit a stack trace (default :0)
       --log_dir string                              If non-empty, write log files in this directory
       --log_file string                             If non-empty, use this log file
+      --log_file_max_size uint                      Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited. (default 1800)
       --logtostderr                                 log to standard error instead of files (default true)
       --metric-blacklist string                     Comma-separated list of metrics not to be enabled. The whitelist and blacklist are mutually exclusive.
       --metric-whitelist string                     Comma-separated list of metrics to be exposed. The whitelist and blacklist are mutually exclusive.
       --namespace string                            Comma-separated list of namespaces to be enabled. Defaults to ""
       --port int                                    Port to expose metrics on. (default 80)
       --skip_headers                                If true, avoid header prefixes in the log messages
-      --stderrthreshold severity                    logs at or above this threshold go to stderr
+      --skip_log_headers                            If true, avoid headers when openning log files
```

Comment:
```suggestion
      --skip_log_headers                            If true, avoid headers when opening log files.
```

Reply:
Ah I see, it was using the release, instead of master, let me fix this!

Reply:
@tariq1890  Actually! I take that back, it's correct, it comes from klog, it's a typo there. If you do `kube-state-metrics --help` you will see it there :) 

So if we want to use the `make generate` to update these docs, we need to keep it as is.

Reply:
See https://github.com/kubernetes/klog/pull/62 :)

Reply:
Thanks !